app/tasks/brief_tasks.py

```diff
@@ -0,0 +1,15 @@
+from . import celery
+from app import db
+from app.models import Brief
+from app.emails.briefs import send_brief_closed_email
+import pendulum
+
+
+@celery.task
+def process_closed_briefs():
+    # find briefs that were closed yesterday. this task is designed to run after midnight.
+    closed_briefs = db.session.query(Brief).filter(Brief.closed_at >= pendulum.yesterday(tz='Australia/Sydney'),
+                                                   Brief.closed_at < pendulum.today(tz='Australia/Sydney')).all()
```

Comment:
Is it possible these filters might pick up withdrawn briefs as well? I imagine a buyer probably doesn't need to be informed when a withdrawn brief has closed. I've previously used `Brief.withdrawn_at.is_(None)` for this. Also, looking at my last brief query, I had `Framework.slug == 'digital-marketplace'` to make sure we are only processing DM briefs (not sure if that's relevant here).

Reply:
added the filters